### PR TITLE
Update Duckquill to v4.2.1

### PR DESCRIPTION
**Pull request by daudix**
Hi, sorry for kicking in like that :sweat_smile: 

I've noticed that your site uses an outdated Duckquill v1.0, so I thought I could help and update it to a more recent v4.2.1; I have made the necessary changes to make it compatible both with the new Duckquill and Zola, I have also added some custom styles in `custom.scss` to make headings look more like in current version.

Screenshot 2024-08-06 at 20-07-50 Bano's Nook
Screenshot 2024-08-06 at 20-08-03 Right now I am  - Bano's Nook

P.S. Блогпост "Тропинки против оградок" мне очень понравился)

### CODE REVIEW

File: content/_index.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ I'm a full-stack developer. Mainly write in TypeScript and passionate about the
 
 You can call me Ivan or _Ваня_ if you can read the latter 😆
 
-To **contact** me you can mention me on <a href="https://mastodon.ml/@bano" rel="me">Mastodon</a>
-or send me an <a href="mailto:me@bano.dev" rel="me">Email</a>.
-You also can find my code on <a href="https://codeberg.org/bano" rel="me">Codeberg</a> or <a href="https://github.com/banonotit" rel="me">GitHub</a>.
+To **contact** me you can mention me on [Mastodon](https://mastodon.ml/@bano)
```
Comment (BANOnotIT) [inline content/_index.md:10]: I'm not sure if mastodon will accept links without rel=me to do verification.

File: content/links.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
 +++
 title = "Links to things I like"
-date = 2024-02-26
-updated = 2024-02-26
-template = "blog.html"
```
Comment (BANOnotIT) [inline content/links.md]: No dates support in new version?..

**Review by BANOnotIT — COMMENTED** [review]
Hello! Thank you for your theme and for the effort to update it for the consumers!

Sadly, I can't merge it right now. I in some magical way am blocked by the fact that nixos-unstable only has zola v0.18. Will have to wait until it's available.

Пы.Сы. Спасибо ;3

File: content/_index.md (hunk 1 above)
Comment (daudix) [reply content/_index.md:10]: Mastodon verification is in fact builtin in Duckquill; once you set comments host and username it gets added in the `<head>` automatically.

Comment (BANOnotIT) [reply content/_index.md:10]: Oh. So you're using `<link>` approach instead of `<a>` approach? Great!

**Pull request closed without merging**